src/ui/components/views/Home.web.js

```diff
@@ -3,15 +3,20 @@
 import React, { Component, PropTypes } from 'react';
 import Radium from 'radium';
 import Colors from '../../Colors';
+import fs from 'fs';
+
+
+const promo = fs.readFileSync(__dirname + '/../../../../templates/promo.html');
```

Comment:
@aravindet Rename the HTML to JS, and export the html as a multiline string from there should work.